src/main/java/seedu/duke/common/InfoMessages.java

```diff
@@ -33,23 +34,30 @@ public enum InfoMessages {
     INFO_STATS_HABIT_VERY_HIGH_SAVINGS("Excellent! You saved quite a lot this month."),
     INFO_STATS_HABIT_HIGH_SAVINGS("Wow, keep up the good work. You saved at least two-third of your income."),
     INFO_STATS_HABIT_MEDIUM_SAVINGS("Good effort, you saved at least half of your income."),
-    INFO_STATS_HABIT_LOW_SAVINGS("Hmm, you spent quite a sum. Try saving more in future."),
-    INFO_STATS_HABIT_VERY_LOW_SAVINGS("Oops, you spent too much. Do manage your expenses within your constraints."),
-    INFO_STATS_TIME_INSIGHTS("Here are the categorical savings and monthly time insights for"),
-    INFO_STATS_CATEGORIES_HEADER("-----CATEGORIES-----"),
-    INFO_STATS_EXPENDITURE_HEADER("-----EXPENDITURE-----"),
+    INFO_STATS_HABIT_LOW_SAVINGS("Hmm, you are spending more than 50% of what you earned. "
```

Comment:
Great change to make the text in stats spending habit much clearer to understand.